localization/english/wc_effects_l_english.yml

```diff
@@ -80,4 +80,16 @@
  raise_undead_by_vassal_effect_third_past:1 "[CHARACTER.GetShortUIName|U] became [GetTrait('being_undead').GetName(GetNullCharacter)|V] and converted to [TARGET_CHARACTER.GetFaith.GetName|V]"
 
  raise_undead_by_vassal_effect_global:1 "Becomes [GetTrait('being_undead').GetName(GetNullCharacter)|V] and converts to [TARGET_CHARACTER.GetFaith.GetName|V]"
- raise_undead_by_vassal_effect_global_past:1 "Became [GetTrait('being_undead').GetName(GetNullCharacter)|V] and converted to [TARGET_CHARACTER.GetFaith.GetName|V]"
\ No newline at end of file
+ raise_undead_by_vassal_effect_global_past:1 "Became [GetTrait('being_undead').GetName(GetNullCharacter)|V] and converted to [TARGET_CHARACTER.GetFaith.GetName|V]"
+ 
+ break_union_effect_global:0 "The [Concept('union', CHARACTER.Custom( 'GetPolFactionName' ))|E] falls apart"
+ break_union_effect_global_past:0 "The [Concept('union', CHARACTER.Custom( 'GetPolFactionName' ))|E] fell apart"
+
+ leave_union_tooltip_first:0 "You leave the current [union|E]"
+ leave_union_tooltip_first_past:0 "You left the current [union|E]"
+ 
+ leave_union_tooltip_third:0 "[CHARACTER.GetShortUIName|U] leaves the current [union|E]"
+ leave_union_tooltip_third_past:0 "[CHARACTER.GetShortUIName|U] left the current [union|E]"
```

Comment:
I suggest to put them in the same file for the sake of convenience

<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46576860/154790863-80c1af6e-cb78-4bd4-ba23-73843e816e37.png)


</details>

Reply:
Also, it has the same problem - since you speak of it in the past tense, it's not longer "current" union.

Reply:
Fixed in bd96e8aba6d3197bd326bb1cefaf5d4cfd25d734.